frontend/app/pages/panel/team.vue

```diff
@@ -0,0 +1,94 @@
+<script setup lang="ts">
+const { data: team, refresh } = await useAuth('/teams/membership/my_team', {
+  key: 'teams-membership-my-team',
+  method: 'GET',
+})
+
+if (!team.value) {
+  await navigateTo('/panel/')
+}
+
+const { _, error } = await useAuth('/teams/management/', {
+  key: 'is-user-leader',
+  method: 'GET',
+})
+
+const is_user_leader = error.value === undefined
+
+const inviteUserModal = ref(false)
+const leaveTeamModal = ref(false)
+const deleteTeamModal = ref(false)
+
+const members = computed(() => {
+  const teamMembers = team?.value?.members || []
+  return [...teamMembers, ...Array.from({ length: 5 }).fill(null)].slice(0, 5)
+})
+
+async function kick(username: string) {
+  const { error } = await useAuth('/teams/management/kick_user', {
+    key: 'teams-management-kick-user',
+    method: 'DELETE',
+    body: {
+      username,
+    },
+  })
+
+  const toast = useToast()
+
+  if (error.value?.data) {
+    const response = error.value.data as any
```

Comment:
I think it would be nice if we could create component, for confirming and eaisly use it here and in other places...